### PR TITLE
Add documentation for On-demand notifications

### PR DESCRIPTION
**Pull request by jmurphy45**
Add documentation for on-demand notifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,17 +20,21 @@ return MicrosoftTeamsMessage::create()
...
-- [Contributing](#contributing)
-- [Credits](#credits)
-- [License](#license)
+- [Microsoft Teams Notifications Channel for Laravel](#microsoft-teams-notifications-channel-for-laravel)
+  - [Contents](#contents)
+  - [Installation](#installation)
+    - [Setting up the Connector](#setting-up-the-connector)
+    - [Setting up the MicrosoftTeams service](#setting-up-the-microsoftteams-service)
+  - [Usage](#usage)
+  - [On-Demand Notification Usage](#on-demand-notification-usage)
```
Comment (Tob0t) [inline README.md:29]: Please move this to a sub item of [Usage]

File: README.md [hunk 2]
```diff
@@ -20,17 +20,21 @@ return MicrosoftTeamsMessage::create()
...
+    - [Available Message methods](#available-message-methods)
```
Comment (Tob0t) [inline README.md:30]: should be the same level as On-Demand Notification usage

File: README.md [hunk 3]
```diff
@@ -117,6 +121,16 @@ public function routeNotificationForMicrosoftTeams(Notification $notification)
 }
 ```
 
+## On-Demand Notification Usage
```
Comment (Tob0t) [inline README.md:124]: Please set to level 3 (###)

**Review by Tob0t — COMMENTED** [review]
just some minor formatting things

File: README.md (hunk 2 above)
Comment (jmurphy45) [reply README.md:30]: Made changes

File: README.md (hunk 3 above)
Comment (jmurphy45) [reply README.md]: Made changes

**Pull request merged**